### PR TITLE
Changing the community links

### PR DESCRIPTION
**Pull request by grctest**
Switch to focusing on the gridcoin.us website instead of limiting chat/exchange selection.

### CODE REVIEW

File: src/qt/bitcoingui.cpp [hunk 1]
```diff
@@ -1521,8 +1521,7 @@ void BitcoinGUI::bxClicked()
 
 void BitcoinGUI::chatClicked()
 {
-    //QDesktopServices::openUrl(QUrl("https://kiwiirc.com/client/irc.freenode.net:6697/#gridcoin"));
-    QDesktopServices::openUrl(QUrl("https://kiwiirc.com/client/irc.freenode.net:6697/#gridcoin-help"));
+    QDesktopServices::openUrl(QUrl("https://gridcoin.us/contact.htm#GridcoinWallet"));
```
Comment (denravonska) [inline src/qt/bitcoingui.cpp]: Shouldn't this point to Slack?

**Comment by grctest** [comment]
@denravonska I got a notification asking about why the 'chat' button shouldn't be slack, but I can't see the message on github.

There are a couple reasons:
* Slack invite link may change soon (official perma slack link - awaiting support, vs herokuapp link).
* Not everyone likes to use slack.
* The contact page provides many options to interact with the gridcoin community, rather than focusing on a single chat program.
* Analytics.

**Comment by denravonska** [comment]
@grctest Yeah, I removed it. The actual link had more than enough forward pointers to cover it :)

**Comment by grctest** [comment]
Changed the changelog as requested.

**Pull request merged**